### DISCUSSION TITLE
Play notification after rendering not working.

### DISCUSSION THREAD

**Discussion by Redtash1** [mashb1t/Fooocus#48] [Q&A] [answered]
Play notification after rendering not working but I figured it out. I was looking through code and config files and saw that it was looking for notification.mp3 and the file in fooocus folder was notification-example.ogg, so I converted it to .mp3 and named it notification.mp3 and it works now. Of course you can make your own notification and name it notification.mp3. It works but I noticed 
that I'm getting these errors now.  Here are pics to show before and after. 
![1](https://github.com/mashb1t/Fooocus/assets/175253144/9d06e280-97ad-4f2a-94fc-ade83f33d0e1)
![2](https://github.com/mashb1t/Fooocus/assets/175253144/b47eabab-dc92-4553-8527-2402e3411057)
![3](https://github.com/mashb1t/Fooocus/assets/175253144/76011de3-e27b-4f47-aa16-1275a19fba0f)
![4](https://github.com/mashb1t/Fooocus/assets/175253144/33914e4b-725d-4cf6-8a71-1b44889e68f0)
![Fooocusv2 5 2 Errors for notifation](https://github.com/mashb1t/Fooocus/assets/175253144/aa92ffa8-6321-4e8d-a874-865a146664f3)

**Comment by mashb1t** [comment, marked as answer]
Gradio needs an initial file to show the audio element correctly, so you need to have any audio first in order to be able to use it.

**Reply by Redtash1** [reply]
Sorry I don't know what that means? I don't know much about coding but I would like to learn. I know just enough to be dangerous 😁😁😁. Thank you.

**Reply by mashb1t** [reply]
The ffmpeg warning can be ignored. To use your own audio file on any browser other than the default you need to add a default audio file in Fooocus.

**Reply by Redtash1** [reply]
I'm using Fooocus in my default browser. Yes I figured since it did work after I renamed file that ffmpeg warning wasn't a problem but I like to know what causes the warning or errors and fix them.  The Audio tab isn't in the original Fooocus, did you add it in your fork? Does the notification work for you? I copied your fork straight from your GitHub installed and everything worked except notification. I used some code to modify it to use Intel Arc GPU do you think it could have changed something? But here is the original code that says it's looking for "notification.mp3" file, unless I'm reading wrong but when I changed the file in Fooocus folder from "notification-example.ogg" to "notification.mp3" it started working. Here is file location C:\Users\DVR\Desktop\Fooocusv2.5.2\Fooocus\webui.py. Thank you. 

![Fooocus Notifation Code](https://github.com/user-attachments/assets/eb981c92-e604-465d-b414-429bf6878f12)

**Reply by mashb1t** [reply]
You don't locally have ffmpeg installed and Gradio tries to stream the audio with specific codecs afaik, which is them causing the warning. This should have nothing to do with your other adjustments.
Feel free to dig deeper yourself, this tab is specific to my fork.

**Reply by Redtash1** [reply]
Ok thank you. Does the notification play for you without renaming the notification.ogg file? Reason I'm asking  all these questions is because LORD willing I'm thinking of making a YouTube video showing how to download your version of Fooocus and installing it on a Window with an Intel Arc GPU and i want to show how to make everything work.  Thank you

**Reply by mashb1t** [reply]
The notification has to be renamed in order to work. I'll add a new notification-example.mp3 file today.

**Reply by Redtash1** [reply]
Ok. Thank you.

**Reply by mashb1t** [reply]
https://github.com/mashb1t/Fooocus/commit/02b06ccb33b81eed17ff76ff01ffb5380a4af210

**Reply by Redtash1** [reply]
Thank you but the file name needs to be notification.mp3 or change the code to notification-example.mp3. Sorry. Thank you.

**Reply by mashb1t** [reply]
this is intended as not everybody wants to have this, but i might rework the default behaviour

**Reply by Redtash1** [reply]
Oh ok. Sorry.
